pyomo/core/tests/unit/test_model.py

```diff
@@ -879,10 +879,9 @@ def c(b):
                           ['y','I','x','c'] )
         self.assertEqual( len(list(EXPR.identify_variables(instance.c.body))), 3 )
 
-    def test_error1(self):
-        model = ConcreteModel()
-        model.x = Var()
-        instance = model.create_instance()
+    def test_error_creating_model_baseclass(self):
+        with self.assertRaises(TypeError):
```

Comment:
It would be better if this were an `assertRaisesRegex` to verify that the "right" `TypeError` was raised.